controllers/custom/builder.go

```diff
@@ -160,6 +166,7 @@ func (b *Builder) Complete(reconciler Reconciler) (healthz.Checker, error) {
 
 				case cache.Deleted:
 					if err := b.dataStore.Delete(convertedObj); err != nil {
```

Comment:
Although I don't think we must have a log here, I recommend we add a simple log string in this case block to log out the deleted object name and ns again.